### DISCUSSION TITLE
Only debug information of the last created apollo client is shown.

### DISCUSSION THREAD

**Discussion by UchihaYuki** [apollographql/apollo-client-devtools#403] [Ideas]
**Intended outcome:**Show debug information for all apollo client instances.

**Actual outcome:**Only debug information for the last created apollo client.

**How to reproduce the issue:**Create more than one apollo client through Apollo Angular.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10
 - Browser [e.g. chrome, safari] chrome
 - Browser version [e.g. 22] Version 88.0.4324.150 (Official Build) (64-bit)
 - Extension version [e.g. 3.0] 2.3.5

**Comment by hwillson** [comment]
Thanks @UchihaYuki - this is a known / intentional limitation. We have plans to change this in the future. I'll move this to a discussion so we don't lose site of the idea. Thanks!
